### ISSUE TITLE
Official Fatiando Mastodon account?

### ISSUE THREAD

**Issue by leouieda** [fatiando/community#75]
Well, since it actually happened and a lot of people are leaving twitter should we create a Fatiando Mastodon account? Like with LinkedIn, we would mostly post the same things on all our social media so it's not that much more work.

Possible server is: https://fosstodon.org/about/more Only problem is that they only allow toots in English because of moderation. We don't really post much in other languages so maybe it's not a big issue. Or it is and we can look for another server (suggestions welcome). 

cc @santisoler @MGomezN @aguspesce

**Comment by santisoler** [comment]
I would love to have a Mastodon account for Fatiando. I know I'm not the most diligent community manager and it's very likely that I'm not going to be the one that have more toots through it. So I'm in favour, but only if it's not a huge load on the rest of the team.

Regarding the instance, I think Fosstodon sounds good. It's sad that they only support English, but they allow exceptions:
> There are exceptions to this rule. All we really ask is the initial Toot is posted be in English. By default, people don’t see replies in their local timelines. If you end up speaking German in DM, or in replies, so what? As long as that conversation doesn’t breach our CoC we’re all good.

Source: https://hub.fosstodon.org/why-fosstodon-is-english-only/

So I don't think we will have any issue if we toot in English and add a reply with a translation or even if we interact with other people in different languages through replies.

Another option would be https://floss.social. But they have way less active users so probably an account in Fosstodon will draw more attention. I'm just mentioning for the record.

**Comment by MGomezN** [comment]
I'm so lost on an Island that I barely have an idea of what is going on. I can see Mastodon as a trending topic and newspapers highlighted massive migration from Twitter.

 I would ask these:

Which are the "principles", "morals", "ethics ", and "values ", that Fatiando represents?
The current version of Twitter is against those principles?

Then yes. Absolutely yes. Even if is more work. In this scenario, the Twitter account will be closed? When?

**Comment by leouieda** [comment]
Thanks for the great points, everyone! I like the plan of sticking with fosstodon, posting in English, and perhaps replying in other languages. We could always post the main Fatiando account in English and share from our personal accounts in other languages (even in language specific servers, like @santisoler has).

> Which are the "principles", "morals", "ethics ", and "values ", that Fatiando represents?

That’s a great question, @MGomezN. I think we all know intrinsically what those are but we’ve never really got them all in writing. We definitely should and it should be widely advertised on our website. Would you mind opening an issue in this repository about putting these things in writing somewhere (I want you to have the credit for the suggestion 🙂)?

> The current version of Twitter is against those principles?

Perhaps not explicitly. At least not yet. So maybe we don’t need to shut that account down just now. Even if a platform is bad we can still use it for good to promote our project and values and attract people to the light of open-everything. At least that’s how I feel about things like ResearchGate and LinkedIn.

> So I'm in favour, but only if it's not a huge load on the rest of the team.

It shouldn’t really be much more work. We don’t post that regularly and just tend to post the same thing on all platforms.

Alright, I’ll make an account on fosstodon and share the log in with whoever wants to give it a try. That then reminds me to write down some instructions for sharing passwords and keeping accounts secure.

**Comment by leouieda** [comment]
Alright, requested an invite to fosstodon.org with the user name `fatiando`. Waiting to see if they let us in 🤞🏾 I'll share the login once we have confirmation.

**Comment by santisoler** [comment]
Adding to @leouieda:

> Even if a platform is bad we can still use it for good to promote our project and values and attract people to the light of open-everything. At least that’s how I feel about things like ResearchGate and LinkedIn.

I like to read (and listen) to what [Cory Doctorow](https://en.wikipedia.org/wiki/Cory_Doctorow) thinks about this *not-so-benevolent social media*. Even if we don't like them because they are centralized, they have shady ways of show your feed, have ads, are run by billionares, etc (pick your rant), they still are a public space in dispute. Yes, they are walled gardens, but there are also people in there having social interactions, and this is not going to go away just because the company behind it is _bad_. I would love to see science to migrate to an open media (like Mastodon), but I still think we can have a good use of the mainstream closed ones to promote the things we care about. Otherwise we will live inside our echo chamber, completely isolated from the rest of the world.

As Cory puts it: the main problem with these monopolistic platforms is they put a very high **switching cost**. If you are not longer comfortable in that platform you could move to another one, but you leave all your interactions behind. There's no way to make a Twitter user to get a post from someone from another platform, therefore if you want Twitter users to read your content you need to have a Twitter account.

With Mastodon this is different: it relies on an **open protocol** that offers no obstacle to read content from other users. You could use any software that uses that protocol (Mastodon is just one of them), publish content and we in Mastodon will be able to read it and interact with you. Plus you could choose the Mastodon server that better suits you need or even run your own if you like it.

Cory fights for legislation that would force these big monopolies to have **interoperability** (we could interoperate between the different social media) in order to lower the switching cost and also make the market more competitive and not ruled by just five Nazgul.

In conclusion, I think we should keep the Twitter account for now, but I would love to promote Mastodon through it.

**Comment by leouieda** [comment]
Done: https://fosstodon.org/@fatiando
